schemas/twitter.js

```diff
@@ -0,0 +1,167 @@
+                limit: { type: GraphQLInt }
+            },
+            //        passing integer 'id' here doesn't work surprisingly, had to use 'id_str'
+            resolve: ({ id_str: tweetId }, { limit = 5 }) => twitter.getRetweets(tweetId, limit)
+        }
+    })
+});
+
+let RetweetType = new GraphQLObjectType({
+
+    name: 'Retweet',
+    description: 'Retweet of a tweet',
+    fields: () => ({
+        id: { type: GraphQLID },
+        created_at: { type: GraphQLString },
+        in_reply_to_tweet_id: {
+            type: GraphQLString,
+            resolve: ({ in_reply_to_status_id }) => in_reply_to_status_id
+        },
+        in_reply_to_user_id: { type: GraphQLInt },
+        in_reply_to_screen_name: { type: GraphQLString },
+        retweeted_status: {
+            type: TweetType
+        },
+        user: {
+            type: UserType
+        }
+    })
+});
+
+let twitterType = new GraphQLObjectType({
+
+    name: 'TwitterAPI',
+    description: 'The Twitter API',
+    fields: {
+        user: {
+            type: UserType,
+            args: {
+                user_id: {
+                    description: 'ID of user account',
+                    type: GraphQLInt
+                },
+                screen_name: {
+                    description: 'Screenname of the user',
+                    type: GraphQLString
+                }
+            },
+            resolve: (_, { user_id, screen_name }) => {
+
+                const { getUser } = twitter;
+
+                if (!user_id && !screen_name) {
+                    return getUser('user_id', 9533042);
+                }
+                else {
+
+                    if (isNaN(parseInt(user_id))) {
+                        return getUser('screen_name', screen_name);
+                    }
+                    else {
+                        return getUser('user_id', user_id);
+                    }
+                }
+            }
+        },
+        tweet: {
+            type: TweetType,
+            args: {
+                id: {
+                    type: new GraphQLNonNull(GraphQLString),
+                    description: 'Unique ID of tweet'
+                }
+            },
+            resolve: (_, { id: tweetId }) => twitter.getTweet(tweetId)
+        },
+        search: {
+            type: new GraphQLList(TweetType),
+            description: "Returns a collection of relevant Tweets matching a specified query.",
+            args: {
+                q: {
+                    type: new GraphQLNonNull(GraphQLString),
+                    description: "A UTF-8, URL-encoded search query of 500 characters maximum, including operators. Queries may additionally be limited by complexity."
+                },
+                count: {
+                    type: GraphQLInt,
+                    description: "The number of tweets to return per page, up to a maximum of 100. Defaults to 15. This was formerly the “rpp” parameter in the old Search API."
```

Comment:
This description has been directly c/c from Twitter API doc. So maybe I'd rather remove limit from doc only as twitter handles this automatically..